vm/src/bytecode.rs

```diff
@@ -83,9 +83,15 @@ pub enum Instruction {
     JumpIf {
         target: Label,
     },
+    JumpIfOrPop {
```

Comment:
Why is this extra jump if bytecode required? Can't we do all with JumpIfTrue and JumpIfFalse?

Reply:
Yes, I think so.

JumpIfOrPop == Dup, JumpIf, Pop?

I'm just copying the implementation from cpython here. How similar to we want our bytecode to be?